src/natcap/invest/sdr/sdr.py

```diff
@@ -804,16 +804,18 @@ def ls_factor_function(percent_slope, flow_accumulation, avg_aspect):
             beta[big_slope_mask] / (1 + beta[big_slope_mask]))
         m_exp[~big_slope_mask] = m_table[m_indexes]
 
+        # from McCool paper: "as a final check against excessively long slope
+        # length calculations ... cap of 333m"
+        # from Rafa, this should really be the upstream area capped to
+        # "333m^2" because McCool is 1D
```

Comment:
Wait, should it be capped to `333**2 m^2` as in the code below or to `333 m^2` as in the comment above?

Reply:
Nice catch. It should be 333^2 m^2. That'd be a dammed confusing comment for a future engineer. Fixed in fc2cebe6bf3357e68b8f5383e3adbaa9c3fbef74